### PR TITLE
Displays the default README, if any

### PR DESCRIPTION
**Pull request by eduardobs79**
Consumes the new property added to GitHub response.

### CODE REVIEW

File: client/components/git-readme.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import React from 'react'
+
+export const GitReadme = React.createClass({
+  propTypes: {
+    github: React.PropTypes.object.isRequired
+  },
+  render () {
+    return (
+      <div className="panel-module" dangerouslySetInnerHTML={{__html: this.props.github.readme}} />
```
Comment (Georgette) [inline client/components/git-readme.js]: What are you thoughts on using `dangerouslySetInnerHTML={{__html: this.props.github.readme}}` versus rendering inside a `<p>{this.props.github.readme} </p>`

**Comment by matt-oc** [comment]
Do we not already have the readme coming in the npm package? npm.readme

**Comment by eduardobs79** [comment]
We still need to use either a local or a remote markdown service.

**Comment by mcdonnelldean** [comment]
Is this good to go?

**Pull request merged**